### PR TITLE
fix up content about profiles+config in aws guide

### PR DESCRIPTION
**Pull request by susanev**
fixes https://github.com/pulumi/pulumi-hugo/issues/1644

### CODE REVIEW

File: themes/default/content/docs/get-started/aws/begin.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ $ export AWS_SECRET_ACCESS_KEY=<YOUR_SECRET_ACCESS_KEY>
 {{% /choosable %}}
 {{< /chooser >}}
 
-For additional information on setting and using AWS credentials, see [AWS Setup]({{< relref "/registry/packages/aws/installation-configuration" >}}).
+For information about AWS profiles, and configuration, see [AWS Setup]({{< relref "/registry/packages/aws/installation-configuration" >}}).
```
Comment (cnunciato) [inline themes/default/content/docs/get-started/aws/begin.md:102]: Couldn’t we keep the same language and just also mention profiles?

```suggestion
For additional information on setting and using AWS credentials or profiles, see [AWS Setup]({{< relref "/registry/packages/aws/installation-configuration" >}}).
```

Comment (susanev) [reply themes/default/content/docs/get-started/aws/begin.md:102]: @cnunciato what additional information is on that page that doesn't include profiles or credentials?

Comment (susanev) [reply themes/default/content/docs/get-started/aws/begin.md:102]: @cnunciato do you specifically not want me to mention config? what credential information is there that's not in the guide and not in the profile content?

**Comment by dixler** [comment]
Hey, @susanev we should mention a code fence with
```
$ aws configure
```
in it in order to imply that if they've set up their machine to work with the AWS CLI or AWS SDK, pulumi will work as well(like other tools they're used to)

I made this suggestion in #1729 and didn't see this PR.

**Comment by dixler** [comment]
@susanev sure! I have #1721 which condenses the `get-started`'s into single pages so we should probably review that first before changing the content.

**Pull request closed without merging**